src/components/top-bar/_top-bar.scss

```diff
@@ -73,6 +73,15 @@
 .player.ad-break,
 .player.menu-active {
   &:not(.overlay-active) {
+    :global(.#{$video-filter}) {
+      &:before {
+        @include bottom-bar-gradient();
```

Comment:
for my understanding, how does it recognize `bottom-bar-gradient` in top-bar if it is not being imported?

Reply:
i didn't actually check after just confirming that it works, but it probably works because both of them are later imported in style.scss, and/or because during runtime they're both part of one css document